src/wizard/wizard.scss

```diff
@@ -27,68 +26,98 @@
     color: t(iui-color-background-positive);
   }
 
+  // Completed steps
+  .iui-wizard-circle {
+    align-items: center;
+    border-radius: 50%;
+    display: flex;
+    width: $iui-l;
+    height: $iui-l;
+    justify-content: center;
+    overflow-wrap: break-word;
+    padding: 0;
+    position: relative;
+    text-align: center;
```

Comment:
padding, position and maybe text-align can all be removed.

(Duplicating comment since code moved)

Reply:
Removed in next push.